src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1907,7 +1907,7 @@ protected function preparePathInfo()
             return '/';
         }
 
-        return (string) $pathInfo;
+        return $pathInfo;
```

Comment:
`$pathinfo` comes from `substr` that returns `string|false` (it can returns an undocumented `null` on some old versions but not in this case since we always make a valid call). If `false` is returned we return in the if above, so `$pathInfo` is always a string here.